client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -135,6 +139,7 @@ class VerificationPortalForm extends Component {
 							/>
 						</div>
 					</div>
+					<Spacer height="15px" />
```

Comment:
This space is 66px

<img width="369" alt="screen shot 2018-07-03 at 2 41 11 pm" src="https://user-images.githubusercontent.com/6378435/42241148-2bfd9590-7ecf-11e8-97cf-586411ea5863.png">